### ISSUE TITLE
Crash when viewing an HTML5 video or playing HTML5 audio file

### ISSUE THREAD

**Issue by kppullin** [ataranto/CefSharp#12]
Both the latest WinForm and Wpf apps crash when showing/playing HTML audio and video files, or when navigating back to a previous page while the video/audio is playing.  Does this issue exist for others?

I've using a version of ffmpeg built locally from Chromium revision 115322, if that matters or helps diagnose the problem (avcodec-53.dll, avformat-53.dll, and avutil-51.dll  aren't bundled with the binary package - https://github.com/downloads/ataranto/CefSharp/CefSharp-0.9-bin.7z).
##### Links used for testing:
- html5video.org
- http://terrillthompson.com/tests/html5-audio.html

**Comment by ataranto** [comment]
Try using the av*.dll files from this CEF binary download and let me know if you still have the same problem:

http://code.google.com/p/chromiumembedded/downloads/detail?name=cef_binary_1.963.439_windows.zip&can=2&q=

**Comment by kppullin** [comment]
I was mistaken - 'CefSharp-0.9-bin.7z' does include the necessary video and audio files.

I first tested running the included test apps from the package against the audo & video test sites, and the apps crashes.

I then tried to repro the issue using the 'cef_binary' release you linked to, and it crashes there also, so I this looks like a problem with CEF itself and not CefSharp.  I'll try to get the latest CEF to build and see what happens there, but until then I'll close this issue.
